packages/vc-handler-ld-local/src/__tests__/issue-verify-flow.test.ts

```diff
@@ -0,0 +1,203 @@
+const customContext = new Map<string, ContextDoc>([
+  [`https://www.w3.org/2018/credentials/examples/v1`, exampleV1],
+  ['https://sphereon-opensource.github.io/vc-contexts/myc/bedrijfsinformatie-v1.jsonld', bedrijfsInformatieV1],
+])
+
+describe('credential-LD full flow', () => {
+  let didKeyIdentifier: IIdentifier
+  let didLtoIdentifier: IIdentifier
+  let agent: TAgent<IResolver & IKeyManager & IDIDManager & ICredentialIssuer & ICredentialHandlerLDLocal>
+
+  jest.setTimeout(1000000)
+  beforeAll(async () => {
+    agent = createAgent({
+      plugins: [
+        new KeyManager({
+          store: new MemoryKeyStore(),
+          kms: {
+            local: new KeyManagementSystem(new MemoryPrivateKeyStore()),
+          },
+        }),
+        new DIDManager({
+          providers: {
+            'did:key': new KeyDIDProvider({ defaultKms: 'local' }),
+            'did:lto': new LtoDidProvider({
+              defaultKms: 'local',
+              connectionMode: IDidConnectionMode.NODE,
+              network: 'T',
+              sponsorPrivateKeyBase58: '5gqCU5NbwU4gc62be39LXDDALKj8opj1KZszx7ULJc2k33kk52prn8D1H2pPPwm6QVKvkuo72YJSoUhzzmAFmDH8',
+            }),
+          },
+          store: new MemoryDIDStore(),
+          defaultProvider: 'did:key',
+        }),
+        new DIDResolverPlugin({
+          resolver: new Resolver({
+            ...getDidKeyResolver(),
+            ...getUniResolver('lto', { resolveUrl: 'https://lto-mock/1.0/identifiers' }),
+            ...getUniResolver('factom', { resolveUrl: 'https://factom-mock/1.0/identifiers' }),
+          }),
+        }),
+        new CredentialIssuer(),
+        new CredentialHandlerLDLocal({
+          contextMaps: [LdDefaultContexts, customContext],
+          suites: [new SphereonEd25519Signature2018(), new SphereonEd25519Signature2020()],
+          bindingOverrides: new Map([
+            // Bindings to test overrides of credential-ld plugin methods
+            ['createVerifiableCredentialLD', MethodNames.createVerifiableCredentialLDLocal],
+            ['createVerifiablePresentationLD', MethodNames.createVerifiablePresentationLDLocal],
+            // We test the verify methods by using the LDLocal versions directly in the tests
+          ]),
+        }),
+      ],
+    })
+    didKeyIdentifier = await agent.didManagerCreate()
+    didLtoIdentifier = await agent.didManagerImport({
+      provider: 'did:lto',
+      did: LTO_DID,
+      controllerKeyId: `${LTO_DID}#sign`,
+      keys: [
+        {
+          privateKeyHex:
+            '078c0f0eaa6510fab9f4f2cf8657b32811c53d7d98869fd0d5bd08a7ba34376b8adfdd44784dea407e088ff2437d5e2123e685a26dca91efceb7a9f4dfd81848',
+          publicKeyHex: '8adfdd44784dea407e088ff2437d5e2123e685a26dca91efceb7a9f4dfd81848',
+          kms: 'local',
+          kid: `${LTO_DID}#sign`,
+          type: 'Ed25519',
+        },
+      ],
+    })
+  })
+
+  it('works with Ed25519Signature2018', async () => {
```

Comment:
I think all of them should start with 'should'.

Reply:
fixed